### ISSUE TITLE
Authentication

### ISSUE THREAD

**Issue by RodolfoSilva** [hansihe/live_data#20]
Hi @hansihe, thanks for bringing LiveData to life.

I want to know the best way to handle authentication with LiveData?

Do you have any plan to support something like `attach_hook ` and `live_session`?

**Comment by hansihe** [comment]
Yep, this is indeed within scope for the library, but will likely be addressed after the fundamentals are worked on a bit more and rare more solid.

**Comment by RodolfoSilva** [comment]
I'm currently trying to add this functionality to my project.

**Comment by RodolfoSilva** [comment]
My plan is to use LiveData with Flutter [Mirai](https://github.com/Securrency-OSS/mirai) library.

**Comment by hansihe** [comment]
That library looks very cool!

You can pass parameters when opening the livedata, the server side will receive that when mounting. That could be used to pass an auth token or whatever.

As of today there is no support for live sessions or hooks.
